### PR TITLE
docs: OEP-67 automated bundle size checking.

### PR DESCRIPTION
**Pull request by brian-smith-tcril**

### CODE REVIEW

File: oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+******
+
+Accepted
+
+Context
+*******
+
+It is important for users of the Open edX platform that we deliver reasonably sized JavaScript bundles. This provides
+faster load times to all users, and is vital for users with low bandwidth and/or metered connections. To ensure we 
+don't unintentionally increase the size of our JavaScript bundles, we should utilize automated bundle size monitoring.
```
Comment (adamstankiewicz) [inline oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst:14]: [curious] Is it worth mentioning that we have _some_ support for bundle size analytics today insofar that we do generate a bundle size report via the standard Webpack configurations provided by frontend-build which can be viewed locally? E.g., when you run `npm run build`, a `report.html` file is generated in the `dist` directory that lets developers see bundle sizes of JS assets (compressed vs. uncompressed). However, this report is not very discoverable. Exposing similar data via these CI checks makes bundle size a more prominent concern than with the tools Open edX engineers have to work with today.

**Review by adamstankiewicz — APPROVED** [review]
As is, these changes LGTM! Left a few non-blocking additional comments/thoughts.

File: oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst (hunk 1 above)
Comment (arbrandes) [reply oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst:14]: I agree it's a good idea to mention it: I myself had totally forgotten about that report!

[the remark this answers is not in the document]
Comment (brian-smith-tcril) [reply oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst]: added in https://github.com/openedx/open-edx-proposals/compare/73eae79af86499f83d01a126913ee59caed1f72d..9ffa820134819769b9b937af06c4bc6e39c4235c

Comment (brian-smith-tcril) [reply oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst]: added in https://github.com/openedx/open-edx-proposals/compare/73eae79af86499f83d01a126913ee59caed1f72d..9ffa820134819769b9b937af06c4bc6e39c4235c

File: oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+BundleWatch
+===========
+
+`BundleWatch`_ is solely focused on ensuring bundle sizes stay under control. It is actively maintained and used by popular projects such as `bootstrap`_. 
+
+Consequence
+***********
+
+`BundleWatch`_ will be adopted by the Open edX community for automatic JavaScript bundle size monitoring.
```
[the remark this answers is not in the document]
Comment (brian-smith-tcril) [reply oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst]: moved in https://github.com/openedx/open-edx-proposals/compare/73eae79af86499f83d01a126913ee59caed1f72d..9ffa820134819769b9b937af06c4bc6e39c4235c

[the remark this answers is not in the document]
Comment (brian-smith-tcril) [reply oeps/best-practices/oep-0011/decisions/0007-bundle-size.rst]: added in https://github.com/openedx/open-edx-proposals/compare/73eae79af86499f83d01a126913ee59caed1f72d..9ffa820134819769b9b937af06c4bc6e39c4235c

**Pull request merged**